common/lib/protocol-definitions/package.json

```diff
@@ -16,7 +16,7 @@
   "types": "dist/index.d.ts",
   "scripts": {
     "build": "concurrently npm:build:compile npm:lint && npm run build:docs",
-    "build:compile": "concurrently npm:typetests:gen npm:tsc npm:build:esnext",
```

Comment:
This is a temporary hack to get around validating the new SignalType of ExternalDataChange in this exploratory PR.